outputs.tf

```diff
@@ -1,9 +1,9 @@
 output "id" {
-  value       = try(aws_ram_resource_share.subnet_sharing[0].id, null)
+  value       = try(aws_ram_resource_share.subnet_sharing.id, null)
   description = "The ID of the subnet share in resource access manager"
 }
 
 output "arn" {
-  value       = try(aws_ram_resource_share.subnet_sharing[0].arn, null)
+  value       = try(aws_ram_resource_share.subnet_sharing.arn, null)
   description = "The ARN of the subnet share in resource access manager"
 }
```

Comment:
Please add a newline at the end of the file...

Reply:
done